src/contexts/SettingProvider.jsx

```diff
@@ -0,0 +1,19 @@
+import PropTypes from "prop-types";
```

Comment:
Please check AuthContext file to follow,
basically no nee to separate context and provider